iOS/Sample/NetworkViewController.m

```diff
@@ -17,6 +17,8 @@ @implementation NetworkViewController
 - (void)viewDidLoad {
   [super viewDidLoad];
   self.navigationItem.title = @"Network";
+  NSArray *array = @[];
+  [array objectAtIndex:4];
```

Comment:
This is added deliberately to cause a crash. I will remove it once I am done with my final changes.